Doc/library/sched.rst

```diff
@@ -44,16 +44,22 @@ Example::
    ...     print(time.time())
    ...     s.enter(10, 1, print_time)
    ...     s.enter(5, 2, print_time, argument=('positional',))
+   ...     # this runs after 'positional' as enter() is relative
    ...     s.enter(5, 1, print_time, kwargs={'a': 'keyword'})
+   ...     s.enterabs(3, 10, print_time, argument=("first enterabs",))
+   ...     s.enterabs(3, 5, print_time, argument=("second enterabs",))
```

Comment:
```suggestion
   ...     s.enterabs(1650000000, 10, print_time, argument=("first enterabs",))
   ...     s.enterabs(1650000000, 5, print_time, argument=("second enterabs",))
```
I think using a higher number than 3 would be less confusing

Reply:
I would add that digit separators would be useful here.

```python
   ...     s.enterabs(1_650_000_000, 10, print_time, argument=("first enterabs",))
   ...     s.enterabs(1_650_000_000, 5, print_time, argument=("second enterabs",))
```